galaxy_ng/app/models/contentguard.py

```diff
@@ -11,11 +11,13 @@ def permit(self, request):
         """
         Authorize the specified request based on if the request is authenticated.
         """
+        if not (drequest := request.get("drf_request", None)):
```

Comment:
The Walrus `:=` for the first time on our codebase :) 💯 
 
![](https://media.giphy.com/media/XCPnjIUMT06ek/giphy.gif?cid=ecf05e47pl5ki7b1sjnny54qy9jab3fm4sfoodzojci4zs47&rid=giphy.gif&ct=g)